.github/workflows/build_images.yml

```diff
@@ -3,7 +3,8 @@ name: Build Docker images for all OpenSlides services
 on: [push, workflow_dispatch]
 
 env:
-  IMAGE_VERSION: 4.0.0-dev
+  IMAGE_VERSION: 4.0-dev
+  TAG_SUFFIX: -$(date +%Y%m%d)-${GITHUB_SHA::7}
```

Comment:
Please change to -$(date +%Y-%m-%d)-${GITHUB_SHA::7} to get a ISO compliant date format.